docs/docs/cmd/spo/homesite/homesite-set.mdx

```diff
@@ -4,7 +4,7 @@ import TabItem from '@theme/TabItem';
 
 # spo homesite set
 
-Sets the specified site as the Home Site
+Sets the specified site as the home site.
```

Comment:
This is not the same description as the one in the code base.

Reply:
Good spot, updated it to the same description in the code base.